example/src/tests.ts

```diff
@@ -620,3 +620,11 @@ test("canManagePreferences", async () => {
   return true
 });
 
+test("is address in the XMTP network", async () => {
+  const alix = await XMTP.Client.createRandom({ env: "local" });
+
+  const isAddressAvailable = await XMTP.publicCanMessage(alix.address, 'local');
```

Comment:
This name `publicCanMessage` doesn't exist anywhere in the code? Are you sure this passes?